common/index.ts

```diff
@@ -0,0 +1,40 @@
+import AdmZip from 'adm-zip';
+import fs from 'fs';
+import fetch from 'isomorphic-fetch';
+
+export enum ProjectSource {
+	FILE = 'file',
+	URI = 'uri',
+	CLOUD_ID = 'cloudId',
+	PROJECT_JSON = 'project.json',
+	MIT_PROJECT_URI = 'https://projects.scratch.mit.edu'
```

Comment:
Have we decided a code style? If we hand this off to LLK we probably want to use their common code style. Another option may be to use [prettier](https://github.com/prettier/prettier). (I don't think prettier supports LLK's code style.)

An idiom I prefer is "changes should only modify their line". Adding a line inside an object literal at its end for example should not require a change to the line before to add a comma. Specifically for the line here we should require [comma-dangle](https://eslint.org/docs/rules/comma-dangle). I think this is something all js projects should do, it makes code changes more readable in those use cases and it makes overlapping changes less likely to conflict.